File: Programmers/Java/Lv.0/[Lv.0] 짝수의 합/[Lv.0] 짝수의 합_00.java
class Solution {
    public int solution(int n) {
        int answer = 0;

        // 홀수 구하기 (2부터 시작, 2씩 증가)
        for (int i = 2; i <= n; i += 2) {
            // 짝수를 모두 더하기
            answer += i;
        }

        return answer;
    }
}

File: Programmers/Python/Lv.0/[Lv.0] 짝수의 합/[Lv.0] 짝수의 합_00.py
def solution(n):
    answer = 0

    # 짝수 구하기 (2부터 시작, 2씩 증가)
    for i in range(2, n + 1, 2):
        # 짝수를 모두 더하기
        answer += i

    return answer
